lessons/basics/modules.md

```diff
@@ -194,9 +202,28 @@ import List, only: :functions
 import List, only: :macros
 ```
 
+It is also worth noting that import is lexical in scope.
+
+```elixir
+iex> defmodule MyModule do
+...>   def func_a do
+...>     import List, only: [duplicate: 2]
```

Comment:
I have not seen an `import/2` used within a function in any Elixir code, is this something we want to encourage?